### PR TITLE
Feature lwdev 6174 update dependencies after canldes producer task 6001

### PR DESCRIPTION
**Pull request by dmitrysmall**

### CODE REVIEW

File: src/Lykke.Job.CandleHistoryWriter.Repositories/Snapshots/SnapshotCandleEntity.cs [hunk 1]
```diff
@@ -39,9 +39,6 @@ public class SnapshotCandleEntity : ICandle
         public DateTime LastUpdateTimestamp { get; set; }
 
         [Key(10)]
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.CandleHistoryWriter.Repositories/Snapshots/SnapshotCandleEntity.cs]: Can we have problems with already saved data? Can old LastTradePrice appears in TradingOppositeVolume?

File: src/Lykke.Job.CandlesHistoryWriter.Core/Lykke.Job.CandlesHistoryWriter.Core.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.6.2" />
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.CandlesHistoryWriter.Core/Lykke.Job.CandlesHistoryWriter.Core.csproj]: I almost sure that Autofac reference can be deleted as it comes transiently via other own libs

File: src/Lykke.Job.CandlesHistoryWriter/Lykke.Job.CandlesHistoryWriter.csproj [hunk 3]
```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.1" />
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.CandlesHistoryWriter/Lykke.Job.CandlesHistoryWriter.csproj]: Probably the reference can be deleted

File: src/Lykke.Job.CandleHistoryWriter.Repositories/Snapshots/SnapshotCandleEntity.cs (hunk 1 above)
Comment (dmitrysmall) [reply src/Lykke.Job.CandleHistoryWriter.Repositories/Snapshots/SnapshotCandleEntity.cs]: This should not become a problem for we update every service wich is a cosumer of the data as well. The only risk is to update the software prior to trades data migration. The second thing is that here we work around a snapshot serialization/deserialization. It is only needed at shutdown/startup of the job and is rewritten each time.

**Pull request merged**